qiskit/test/base.py

```diff
@@ -110,6 +112,20 @@ def _get_resource_path(filename, path=Path.TEST):
         """
         return os.path.normpath(os.path.join(path.value, filename))
 
+    def assertQuantumCircuitEqual(self, qc1, qc2, msg=None):
+        """Extra assertion method to give a better error message when two circuits are unequal."""
+        if qc1 == qc2:
+            return
+        if msg is None:
+            msg = "The two circuits are not equal."
+        msg += f"""
+Left circuit:
+{qc1.draw(style='text')}
```

Comment:
You can actually just do:

```suggestion
{qc1}
```

as `__str__` for `QuantumCircuit` is defined to do this already: https://github.com/Qiskit/qiskit-terra/blob/main/qiskit/circuit/quantumcircuit.py#L471-L472

Also I'm surprised `style='text'` works because that argument is normally used to control the mpl stylesheet used for the visualization and `output` is the argument used to select between latex, text, mpl, etc.

Reply:
Oh whoops lol.  I basically never use any arguments to `draw` so I'd forgotten what they were called, and since it worked as I expected, I just assumed I'd got it right.

Reply:
Looking at it in detail the drawer defaults to the text drawer and the style argument is just ignored in text mode (and only for text mode).